src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,69 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Represents the response json with the aggregated buildbot data.
+ */
+public class BuildInfo {
+
+  // git commit hash
+  private final String commitHash;
+  // time of receiving build info
+  private final Date timestamp;
```

Comment:
I'm a bit unsure about why would we need this field with the meaning as described here (time of receiving build info). All other bits of information are useful data about the build runs, independent of this application (their source is the buildbots themselves). This however feels more like a transient property of this application (when did we query the buildbots for data), which seems odd.

This may be just me not sure about the intended purpose of this field and it may in fact be important; it's just that I'm unclear.

Note that the Buildbot API contains some timestamps like started_at and completed_at, which are inherent buildbot data: http://docs.buildbot.net/latest/developer/raml/build.html

Additionally, if the meaning of this field is is really the time of receiving the build info, consider making the name more specific, like: `received_at`, instead of `timestamp`. The reason is that the word `timestamp` is way too generic and could, by itself, be interpreted in various ways.

Reply:
We decided to make this field display the time of when the tested commit was pushed

Reply:
Great! As a follow-up from my previous comment, consider renaming this field to something more specific than `timestamp`, e.g., `pushed_at` could be a good option. The reason is that the word timestamp is way too generic and could, by itself, be interpreted in various ways; also this rename would allow for more consistent updates later if we add more timestamp fields to the POJO.